src/PrestaShopBundle/Resources/config/services/bundle/twig.yml

```diff
@@ -58,3 +58,10 @@ services:
             - '@prestashop.static_cache.adapter'
         tags:
             - { name: twig.extension }
+
+    prestashop.bundle.twig.extension.documentation_link_provider_extension:
+      class: 'PrestaShopBundle\Twig\Extension\DocumentationLinkExtension'
+      arguments:
+        - '@prestashop.core.util.helper_card.documentation_link_provider'
```

Comment:
@matks see `documentation_link` below, but basically i think that this service should be moved under `Core/Documentation` namespace since it can be used not only for cards, wdyt?

Reply:
Yes we can 👍 we just need to a service alias to preserve backwards-compatibility

Reply:
Agreed!

Reply:
🆙